model/src/main/kotlin/licenses/LicenseConfiguration.kt

```diff
@@ -19,61 +19,74 @@
 
 package org.ossreviewtoolkit.model.licenses
 
+import com.fasterxml.jackson.annotation.JsonAlias
 import com.fasterxml.jackson.annotation.JsonInclude
 
 /**
- * A configuration for licenses which allows to assign meta data to licenses. It allows assigning licenses to sets which
- * are useful for making license classifications. The available license sets are implicitly defined by the references
- * from the [licenses] and it is optional to define the meta data for any license set via an entry in [licenseSets].
+ * A classification for licenses which allows to assign meta data to licenses. It allows defining rather generic
+ * categories and assigning licenses to these. That way flexible classifications can be created based on
+ * customizable categories. The available license categories need to be declared explicitly; when creating an
+ * instance, it is checked that all the references from the [categorizations] point to existing [categories].
  */
 data class LicenseConfiguration(
     /**
-     * Defines meta data for the license sets.
+     * Defines meta data for the license categories.
      */
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
-    val licenseSets: List<LicenseSet> = emptyList(),
+    @JsonAlias("license_sets")
+    val categories: List<LicenseCategory> = emptyList(),
```

Comment:
Slightly unrelated, as in this regard the code did not change, but shouldn't this better be a `Set` as we would not want duplicate categories being defined? Anyway, this could be changes in a follow-up PR.

Reply:
Agreed, a set would probably be more concise. However, as categories are complex objects, it would still be possible for the set to contain multiple categories with the same name, but different descriptions. We need to manually check for duplicate IDs anyway, so this would not make much of a difference. The same would btw apply to the categorizations, wouldn't it?

Reply:
Maybe I'm missing something, but wouldn't it become impossible to complain to the user about duplicate entries if the de-serialization took care of eliminating those, which is what would happen if Set's were used. Avoiding duplicate categories could be done in the `init` block.

Reply:
> Maybe I'm missing something, but wouldn't it become impossible to complain to the user about duplicate entries

You're right.